stock-usecase/build.gradle.kts

```diff
@@ -92,6 +92,7 @@ dependencies {
     implementation(project(":dhis_android_analytics"))
 
     implementation(libs.bundles.stock.implementation)
+    implementation(project(":aggregates"))
```

Comment:
why stock use case should know aggregates? shouldn't use the table directly?

Reply:
Yep. Not needed as the table will be consumed form the mobile ui